bitbots_vision/bitbots_vision/yoeo_vision.py

```diff
@@ -176,21 +177,18 @@ def _register_subscribers(self, config: Dict) -> None:
             callback=self._image_callback
         )
 
-    def _image_callback(self, image_msg) -> None:
-        # TODO: type
+    def _image_callback(self, image_msg: Image) -> None:
```

Comment:
Same here, does it still run? Most "missing" type hints aren't missing because I forgot them...

Reply:
This one definitely worked on the robot.